tests/common/utilities.py

```diff
@@ -819,6 +819,58 @@ def get_downstream_neigh_type(topo_type, is_upper=True):
     return None
 
 
+def run_until(delay, retry, condition, function, *args, **kwargs):
```

Comment:
Is it better to use argument name `interval` than `delay`? Another thing is that when we want to repeatedly do something, besides interval and retry numbers, we may want some initial delay. So, is it better to add another argument to introduce initial delay? The default value of initial delay can be 0 seconds.

Reply:
@wangxin done